apps/dashboard/app/lib/smart_attributes/attributes/auto_groups.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module SmartAttributes
+  class AttributeFactory
+    # Build this attribute object. No options are used as this Attribute
+    # is meant to be dynamically generated from the users' unix groups
+    # @param opts [Hash] attribute's options
+    # @return [Attributes::AutoGroups] the attribute object
+    def self.build_auto_groups(opts = {})
+      options = begin
+        CurrentUser.group_names.grep(/#{Configuration.auto_groups_filter}/)
+      rescue RegexpError => e
+        Rails.logger.warn("auto_groups_filter does not compile, throwing this error: #{e}")
+        []
+      end
+
+      static_opts = {
+        options: options
+      }.merge(opts.without(:options).to_h)
+
+      Attributes::AutoGroups.new('auto_groups', static_opts)
+    end
+  end
+
+  module Attributes
+    class AutoGroups < Attribute
+      def widget
+        'select'
+      end
+
+      def label(*)
```

Comment:
```suggestion
      def label(*args)
```
I'm not used to seeing a _splat_ without something like this. Is there any difference between the two? I only ask because I almost missed that there was even _splats_ in the method defs.

Reply:
Yes with `*args` you may actually want to reference some thing being passed in. I.e., `args[:that_thing_im_looking_for]`.

With just `*` you can't reference the input arguments, because they don't have a name.  But that's OK because we're not using any of the input parameters anyhow.

So it ends up being a linting issue - why name something you're not using.
![image](https://user-images.githubusercontent.com/4874123/204352578-ec73e741-2507-4ed2-9b36-d39110518100.png)

This is the full message:

>  Unused method argument - `foo`. If it's necessary, use `_` or `_foo` as an argument name to indicate that it won't be used. You can also write as `submit(*)` if you want the method to accept any arguments but don't care about them.rubocop(Lint/UnusedMethodArgument)